recipe/patches/0001-ufunc_object-pass-args-by-reference.patch

```diff
@@ -1,8 +1,17 @@
+From af824dc552c4903b9b17221a06a1350d96e41235 Mon Sep 17 00:00:00 2001
+From: Matti Picus <matti.picus@gmail.com>
```

Comment:
If possible I prefer my mail not appear in the feedstock source code, even though it is pretty easy to find it anyway

Reply:
OK, that's not an issue - sorry, I was not aware of that (I just copied the email from the numpy history).

Reply:
Does that also go for being anywhere in the git history? If so, I'd have to force-remove https://github.com/conda-forge/numpy-feedstock/pull/238/commits/2025e979d84123c87e2ed019fbe614597e848201 - I did some unintentional gymnastics with that just now, but kept it in for now.

Reply:
:shrug: no big deal, thanks